content/node/4b8018dc-49a6-4018-9c54-e8c3e462beee.json

```diff
@@ -98,7 +98,7 @@
                     }
                 ],
                 "lang": "en",
-                "target_revision_id": "2",
+                "target_revision_id": "9df33d0d-239e-4b51-9d97-8703187629b4",
```

Comment:
do you have to manually do this?

Reply:
:( yeah. I need to circle back to this to see if it can be less manual.